lib/parser.ex

```diff
@@ -0,0 +1,198 @@
+defmodule Membrane.Ogg.Parser do
```

Comment:
```suggestion
defmodule Membrane.Ogg.Parser do
  @moduledoc false
```

In Elixir, we mark private modules by adding `@moduledoc false`. This way they are only accessible inside package and they are not visible in documentation (after running mix docs)